src/mockServiceWorker.js

```diff
@@ -74,11 +74,22 @@ self.addEventListener('message', async function (event) {
   }
 })
 
-self.addEventListener('fetch', async function (event) {
```

Comment:
Fetch event handler function cannot be async per specification. Marking it as `async` here is misleading: you cannot use `await` within its top scope, it will cause an invalid state exception (cannot handle fetch event that's already handled).